src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -681,7 +698,6 @@ private static async Task<GraphQLRequest> DeserializeFromGraphBodyAsync(Stream b
         return new GraphQLRequest { Query = query };
     }
 
-#if NET5_0_OR_GREATER
```

Comment:
?

Reply:
Charset is now decoded for any TFM, and used when reading media type application/graphql for any TFM. Since no transcoding stream available, it does not yet affect json reading on older TFMs

Reply:
ok